### ISSUE TITLE
Suggestion: Adobe Acrobat Reader

### ISSUE THREAD

**Issue by evfool** [folkswithhats/fedy#142]
I occasionally have to use Acrobat Reader, as open-source PDF viewers do not support the tax forms I have to fill each month. It would be nice if Fedy would have a way to install Acrobat Reader, as right now after reinstalling I have to search on google/duckduckgo on how to install it on Fedora each time, and fedy is a great tool to save me from this hassle for several applications, like skype and chrome and others.
Please add Acrobat Reader install support too.

**Comment by Rohan62442** [comment]
Adobe no longer supports Adobe Reader on Linux (the last version was 9.5.5) since about 20 months ago. So there will be no guarantee that the latest version will work properly. You will receive no updates either.
I'll look into the plugin if I find reliable packages.

**Comment by evfool** [comment]
I know that Adobe no longer supports Linux, but unfortunately Linux does not support some features of the 20+ months old Adobre Reader. So as long as it works (and it does on F20 and F21 as far as I've tested it) I would be happy to have it.
The instructions I have used are available at http://www.if-not-true-then-false.com/2010/install-adobe-acrobat-pdf-reader-on-fedora-centos-red-hat-rhel/, and Adobe Reader works from that site.

**Comment by satya164** [comment]
What features do you need in particular?

**Comment by palob** [comment]
@evfool Did you try http://mupdf.com/?
It's more capable than poppler-based viewers.

**Comment by evfool** [comment]
@satya164 : form editing is the feature I need (poppler has basic support, but it doesn't work on this document), @palob : mupdf doesn't help either. The [form to be filled](http://static.anaf.ro/static/10/Anaf/Declaratii_R/AplicatiiDec/D392_XML_v102_2014_230215.pdf) is an official one, provided by the Romanian Financial Institute, so I think I will have to live with this.

**Comment by gitgt** [comment]
@evfool: does Master PDF Editor available in Fedy completes the task?

**Comment by Abhinav1217** [comment]
Acrobat reader in not supported in linux anymore. Last update for linux was 2 years ago (according to wikipedia). 
Nowadays PDF can be opened directly in any browser. Master PDF Editor is also a very feature fill application.  
Evince and Google Chrome can work with PDF Forms from government. I use google chrome with `Notable PDF Plugin` for my govt forms. I found about it here. https://www.linux.com/news/software/applications/792447-3-alternatives-to-the-adobe-pdf-reader-on-linux .

Closing this issue.
